docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing
+
+Thank you for thinking about contributing to the WordPress sitemaps project! Contributions can either be made in the form of code or through input on tickets.

```

Comment:
Is it the "WordPress sitemaps project" or "Google XML Sitemaps project" or simply "core sitemaps" ? I'm not totally clear myself.